.github/workflows/publish-image.yaml

```diff
@@ -0,0 +1,51 @@
+on:
+  push:
+    branches:
+    - main
+  workflow_dispatch:
+name: Publish Latest Trickster Image
+
+jobs:
+  publish-image:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: read
+        packages: write
+        attestations: write
+        id-token: write
+    steps:
+      - name: Get Build Date
+        id: builddate
+        run: echo "::set-output name=builddate::$(date -u +%Y-%m-%dT%H:%M:%SZ)"
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        if: github.repository == 'trickstercache/trickster' # skip for forks
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: build & push image
+        run: |
+          b=$(git branch -r --contains ${{ github.ref }}) && branch=${b##*/}
+          docker buildx build \
+            --label "org.opencontainers.image.revision=${{ github.sha }}" \
+            --label "org.opencontainers.image.source=https://github.com/${{ github.repository }}" \
+            --label "org.opencontainers.image.created=${{ steps.builddate.outputs.builddate }}" \
+            --platform linux/arm64/v8,linux/amd64 \
+            -f ./Dockerfile \
+            --push ${{ github.repository == 'trickstercache/trickster' && format('-t {0}:${branch} -t {0}:{1}', env.DOCKER_IO, github.sha) || '' }} \
+            -t ghcr.io/${{ github.repository_owner }}/trickster:${branch} \
+            -t ghcr.io/${{ github.repository_owner }}/trickster:${{ github.sha }} \
+            .
+        env:
+          DOCKER_IO: trickstercache/trickster
```

Comment:
Sorry one more ask here, any chance we could change this env name to `DOCKER_REPO`?

Reply:
done